EIPS/eip-2488.md

```diff
@@ -0,0 +1,58 @@
+---
+eip: 2488
+title: Deprecate the CALLCODE opcode
+author: Alex Beregszaszi (@axic)
+discussions-to: https://ethereum-magicians.org/t/eip-2488-deprecate-the-callcode-opcode/3957
+status: Draft
+type: Standards Track
+category: Core
+created: 2019-12-20
+requires: 7
+---
+
+## Abstract
+
+Deprecate `CALLCODE` in a *somewhat* backwards compatible way, by making it always return failure.
+
+## Motivation
+
+`CALLCODE` was part of the Frontier release of Ethereum. In the first few weeks/months it became clear
+that it cannot accomplish its intended design goal. This was rectified with introducing `DELEGATECALL`
+([EIP-7](https://eips.ethereum.org/EIPS/eip-7)) in the Homestead update (early 2016).
```

Comment:
```suggestion
([EIP-7](./eip-7)) in the Homestead update (early 2016).
```
Relative paths will make the links work in GitHub, forks, branches, mirrors, etc.

Reply:
Absolute paths are the standard required, it has been the case for years.

Reply:
I have been told in the past to use relative paths (I think it was @Arachnid, quite a while back).  Perhaps we should discuss this elsewhere, but is there somewhere I can read to catch-up on any history of discussion on this issue?

Reply:
Over the past 2 years there was an extensive amount of work, mostly by @fulldecent, to change every single link to the absolute. I cannot spend the time right now to dig up the issue numbers.

@Arachnid probably said that probably in 2016-2017. I also preferred the relative paths, but sentiment agreed with the absolute ones.

Reply:
I'd suggest this discussion is handled outside of this pull request.